app/components/form/new-instance.js

```diff
@@ -73,7 +73,10 @@ export default class NewInstanceComponent extends Component {
     notifyFormSavedSuccessfully(this.toaster);
 
     if (this.args.onCreate) {
-      this.args.onCreate(id);
+      this.args.onCreate({
+        instanceTtl: triples,
+        instanceId: id,
+      });
```

Comment:
This is a bit weird, you add arguments that are never used in this version.. (instanceTtl is never used I think).

Reply:
it's done because the onSave for mandataris needs it. It uses the instanceTtl to fetch the period of the mandatee and set their fraction membership to the same period every time (on create and update)